extensions/ql-vscode/src/cli.ts

```diff
@@ -1142,19 +1142,26 @@ export class CodeQLCliServer implements Disposable {
    *   the default CLI search path is used.
    * @returns A dictionary mapping qlpack name to the directory it comes from
    */
-  resolveQlpacks(
+  async resolveQlpacks(
     additionalPacks: string[],
-    searchPath?: string[],
+    extensionPacksOnly = false,
```

Comment:
When `extensionPacksOnly` is false, does it include information about normal qlpacks and extension packs, or only non-extension qlpacks? I'm asking this because we only add `"--kind", "extension"` in the `extensionPacksOnly` case.

I'm wondering if "only" is misleading, and perhaps something like `packType: "qlpack" | "extensionPack"` would be more accurate.

Reply:
The default case is to return all packs. `--kind=extension` will return extension packs only. `--kind=query` for query packs and `--kind=library` will return library packs (note that extension packs are a variant of library packs and will be returned as well).

I can add comments to explain this better.

Reply:
Thanks for explaining. The parameter name and behaviour is fine then 👍